lib/build.rb

```diff
@@ -118,6 +124,8 @@ def query_pull_requests
 
           if p.head.repo.full_name == p.base.repo.full_name
             $logger.info("Skipping pull-request originating from head repo: #{p.number}")
+            @branch_to_pr_number[p.head.ref] = p.number
```

Comment:
This PR check should be called before the branch check.  This function has always skipped "internal" PRs, where the branch lives inside NREL/EnergyPlus.  Now it still does that...but before skipping, it grabs the branch name and PR number to keep them for later.